Cases are not accessible

<!--
Please read the Contributing guidelines (https://github.com/hzi-braunschweig/SORMAS-Project/blob/development/docs/CONTRIBUTING.md) before submitting an issue. You don't have to remove this comment or any other comment from this issue as they will automatically be hidden.
-->
### Bug Description
Several health departments report that some cases are not accessible or changeable anymor. After entering the case the screen is white. The menu left is still visible, the case view and bubbles on the right side are also hide.
It is possible to find the person an the attached case in person search. The link also leads to the blank page.
This issue can be accorded to issue [Reporting Tool] Error during Archive / De-archive cases #10259
The only discrepancy is that it is not possible to send the case to the reporting tool.
It is not possible to test this issue in order to find if another sending to reporting tool has any effect on wrongly displayed case.
Issue could be caused by third-party-application (CISS)

### Steps to Reproduce
1. Cases have been in system
2. Cases has been automatically archived after effective date
3. Try to find person data via person search
4. Result is visible in person search, page is blank, nothing else is accessible

### Expected Behavior
Cases should be always displayed an accessible at any time (even if they are archieved).

### Screenshots
![image](https://user-images.githubusercontent.com/91321821/189359229-ca7cbce5-cbbe-42ea-8ad2-b20444ca6126.png)


### System Details
<!-- Mandatory; you only have to specify the Server URL if the error appeared on a publicly available test server -->
* Device: Unknown
* SORMAS version: 1.74.3
* Android version/Browser: Unknown
* Server URL: Instance for Gummersbach
* User Role: Admin, national user
*

### Additional Information

[Screenshot.pdf](https://github.com/hzi-braunschweig/SORMAS-Project/files/9489547/Screenshot.pdf)
[Screenshot_Archiv.pdf](https://github.com/hzi-braunschweig/SORMAS-Project/files/9489548/Screenshot_Archiv.pdf)

This is the exception that leads to the case not showing. There's an error accessing the `getIndexList` method in `ExternalShareInfoFacade`:

```
{"Timestamp":"2022-09-12T15:28:10.117+0200","Level":"WARNUNG","Version":"Payara 5.2021.10","LoggerName":"javax.enterprise.ejb.container","ThreadID":"36040","ThreadName":"http-thread-pool::http-listener-1(21)","TimeMillis":"1662989290117","LevelValue":"900","MessageID":"AS-EJB-00056","LogMessage":"A system exception occurred during an invocation on EJB ExternalShareInfoFacade, method: public java.util.List de.symeda.sormas.backend.share.ExternalShareInfoFacadeEjb.getIndexList(de.symeda.sormas.api.share.ExternalShareInfoCriteria,java.lang.Integer,java.lang.Integer)"}
{"Timestamp":"2022-09-12T15:28:10.118+0200","Level":"WARNUNG","Version":"Payara 5.2021.10","LoggerName":"javax.enterprise.ejb.container","ThreadID":"36040","ThreadName":"http-thread-pool::http-listener-1(21)","TimeMillis":"1662989290118","LevelValue":"900","Throwable":{"Exception":"Exception thrown from bean: java.lang.NullPointerException","StackTrace":"javax.ejb.TransactionRolledbackLocalException: Exception thrown from bean: java.lang.NullPointerException\n\tat com.sun.ejb.containers.EJBContainerTransactionManager.checkExceptionClientTx(EJBContainerTransactionManager.java:615)\n\tat com.sun.ejb.containers.EJBContainerTransactionManager.postInvokeTx(EJBContainerTransactionManager.java:485)\n\tat com.sun.ejb.containers.BaseContainer.postInvokeTx(BaseContainer.java:4601)\n\tat com.sun.ejb.containers.BaseContainer.postInvoke(BaseContainer.java:2134)\n\tat com.sun.ejb.containers.BaseContainer.postInvoke(BaseContainer.java:2104)\n\tat com.sun.ejb.containers.EJBObjectInvocationHandler.invoke(EJBObjectInvocationHandler.java:212)\n\tat com.sun.ejb.containers.EJBObjectInvocationHandlerDelegate.invoke(EJBObjectInvocationHandlerDelegate.java:79)\n\tat com.sun.proxy.$Proxy686.getIndexList(Unknown Source)\n\tat jdk.internal.reflect.GeneratedMethodAccessor4799.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat com.sun.corba.ee.impl.presentation.rmi.ReflectiveTie.dispatchToMethod(ReflectiveTie.java:143)\n\tat com.sun.corba.ee.impl.presentation.rmi.ReflectiveTie._invoke(ReflectiveTie.java:173)\n\tat com.sun.corba.ee.impl.protocol.ServerRequestDispatcherImpl.dispatchToServant(ServerRequestDispatcherImpl.java:528)\n\tat com.sun.corba.ee.impl.protocol.ServerRequestDispatcherImpl.dispatch(ServerRequestDispatcherImpl.java:199)\n\tat com.sun.corba.ee.impl.protocol.MessageMediatorImpl.handleRequestRequest(MessageMediatorImpl.java:1565)\n\tat com.sun.corba.ee.impl.protocol.SharedCDRClientRequestDispatcherImpl.marshalingComplete(SharedCDRClientRequestDispatcherImpl.java:128)\n\tat com.sun.corba.ee.impl.protocol.ClientDelegateImpl.invoke(ClientDelegateImpl.java:258)\n\tat com.sun.corba.ee.impl.presentation.rmi.StubInvocationHandlerImpl.privateInvoke(StubInvocationHandlerImpl.java:188)\n\tat com.sun.corba.ee.impl.presentation.rmi.StubInvocationHandlerImpl.invoke(StubInvocationHandlerImpl.java:140)\n\tat com.sun.corba.ee.impl.presentation.rmi.codegen.CodegenStubBase.invoke(CodegenStubBase.java:226)\n\tat de.symeda.sormas.api.share.__ExternalShareInfoFacade_Remote_DynamicStub.getIndexList(de/symeda/sormas/api/share/__ExternalShareInfoFacade_Remote_DynamicStub.java)\n\tat de.symeda.sormas.api.share._ExternalShareInfoFacade_Wrapper.getIndexList(de/symeda/sormas/api/share/_ExternalShareInfoFacade_Wrapper.java)\n\tat de.symeda.sormas.ui.externalsurveillanceservice.ExternalShareInfoList.reload(ExternalShareInfoList.java:57)\n\tat de.symeda.sormas.ui.externalsurveillanceservice.ExternalSurveillanceShareComponent.createShareInfoList(ExternalSurveillanceShareComponent.java:94)\n\tat de.symeda.sormas.ui.externalsurveillanceservice.ExternalSurveillanceShareComponent.initLayout(ExternalSurveillanceShareComponent.java:60)\n\tat de.symeda.sormas.ui.externalsurveillanceservice.ExternalSurveillanceShareComponent.<init>(ExternalSurveillanceShareComponent.java:45)\n\tat de.symeda.sormas.ui.externalsurveillanceservice.ExternalSurveillanceServiceGateway.addComponentToLayout(ExternalSurveillanceServiceGateway.java:98)\n\tat de.symeda.sormas.ui.externalsurveillanceservice.ExternalSurveillanceServiceGateway.addComponentToLayout(ExternalSurveillanceServiceGateway.java:44)\n\tat de.symeda.sormas.ui.caze.CaseDataView.initView(CaseDataView.java:183)\n\tat de.symeda.sormas.ui.caze.AbstractCaseView.enter(AbstractCaseView.java:239)\n\tat com.vaadin.navigator.Navigator.performNavigateTo(Navigator.java:778)\n\tat com.vaadin.navigator.Navigator.lambda$navigateTo$9a874efd$1(Navigator.java:702)\n\tat com.vaadin.navigator.ViewBeforeLeaveEvent.navigate(ViewBeforeLeaveEvent.java:54)\n\tat de.symeda.sormas.ui.utils.AbstractDetailView.showUnsavedChangesPopup(AbstractDetailView.java:90)\n\tat de.symeda.sormas.ui.utils.AbstractDetailView.beforeLeave(AbstractDetailView.java:83)\n\tat com.vaadin.navigator.Navigator.runAfterLeaveConfirmation(Navigator.java:730)\n\tat com.vaadin.navigator.Navigator.navigateTo(Navigator.java:701)\n\tat com.vaadin.navigator.Navigator.navigateTo(Navigator.java:678)\n\tat de.symeda.sormas.ui.SormasUI.refreshView(SormasUI.java:100)\n\tat de.symeda.sormas.ui.caze.CaseController.saveCase(CaseController.java:578)\n\tat de.symeda.sormas.ui.caze.CaseController.saveCaseWithFacilityChangedPrompt(CaseController.java:1465)\n\tat de.symeda.sormas.ui.caze.CaseController.lambda$getCaseDataEditComponent$17(CaseController.java:906)\n\tat de.symeda.sormas.ui.utils.CommitDiscardWrapperComponent.onCommit(CommitDiscardWrapperComponent.java:693)\n\tat de.symeda.sormas.ui.utils.CommitDiscardWrapperComponent.doCommit(CommitDiscardWrapperComponent.java:536)\n\tat de.symeda.sormas.ui.utils.CommitDiscardWrapperComponent.commit(CommitDiscardWrapperComponent.java:471)\n\tat de.symeda.sormas.ui.utils.CommitDiscardWrapperComponent.commitAndHandle(CommitDiscardWrapperComponent.java:567)\n\tat de.symeda.sormas.ui.utils.CommitDiscardWrapperComponent$1.buttonClick(CommitDiscardWrapperComponent.java:378)\n\tat jdk.internal.reflect.GeneratedMethodAccessor3804.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat com.vaadin.event.ListenerMethod.receiveEvent(ListenerMethod.java:706)\n\tat com.vaadin.event.EventRouter.fireEvent(EventRouter.java:399)\n\tat com.vaadin.event.EventRouter.fireEvent(EventRouter.java:363)\n\tat com.vaadin.server.AbstractClientConnector.fireEvent(AbstractClientConnector.java:1190)\n\tat com.vaadin.ui.Button.fireClick(Button.java:384)\n\tat com.vaadin.ui.Button$1.click(Button.java:57)\n\tat jdk.internal.reflect.GeneratedMethodAccessor4017.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat com.vaadin.server.ServerRpcManager.applyInvocation(ServerRpcManager.java:155)\n\tat com.vaadin.server.ServerRpcManager.applyInvocation(ServerRpcManager.java:116)\n\tat com.vaadin.server.communication.ServerRpcHandler.handleInvocation(ServerRpcHandler.java:442)\n\tat com.vaadin.server.communication.ServerRpcHandler.handleInvocations(ServerRpcHandler.java:407)\n\tat com.vaadin.server.communication.ServerRpcHandler.handleRpc(ServerRpcHandler.java:275)\n\tat com.vaadin.server.communication.UidlRequestHandler.synchronizedHandleRequest(UidlRequestHandler.java:91)\n\tat com.vaadin.server.SynchronizedRequestHandler.handleRequest(SynchronizedRequestHandler.java:40)\n\tat com.vaadin.server.VaadinService.handleRequest(VaadinService.java:1637)\n\tat com.vaadin.server.VaadinServlet.service(VaadinServlet.java:464)\n\tat javax.servlet.http.HttpServlet.service(HttpServlet.java:750)\n\tat org.apache.catalina.core.StandardWrapper.service(StandardWrapper.java:1637)\n\tat org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:331)\n\tat org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:211)\n\tat de.symeda.sormas.ui.Slf4jMdcFilter.doFilter(Slf4jMdcFilter.java:29)\n\tat org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:253)\n\tat org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:211)\n\tat de.symeda.sormas.ui.SessionFilter.lambda$doFilter$2(SessionFilter.java:73)\n\tat de.symeda.sormas.ui.SessionFilterBean.doFilter(SessionFilterBean.java:37)\n\tat jdk.internal.reflect.GeneratedMethodAccessor3754.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.glassfish.ejb.security.application.EJBSecurityManager.runMethod(EJBSecurityManager.java:588)\n\tat org.glassfish.ejb.security.application.EJBSecurityManager.invoke(EJBSecurityManager.java:408)\n\tat com.sun.ejb.containers.BaseContainer.invokeBeanMethod(BaseContainer.java:4835)\n\tat com.sun.ejb.EjbInvocation.invokeBeanMethod(EjbInvocation.java:665)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:834)\n\tat com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)\n\tat com.sun.ejb.containers.interceptors.SystemInterceptorProxy.doCall(SystemInterceptorProxy.java:163)\n\tat com.sun.ejb.containers.interceptors.SystemInterceptorProxy.aroundInvoke(SystemInterceptorProxy.java:140)\n\tat jdk.internal.reflect.GeneratedMethodAccessor251.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)\n\tat com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)\n\tat org.jboss.weld.module.ejb.AbstractEJBRequestScopeActivationInterceptor.aroundInvoke(AbstractEJBRequestScopeActivationInterceptor.java:72)\n\tat org.jboss.weld.module.ejb.SessionBeanInterceptor.aroundInvoke(SessionBeanInterceptor.java:52)\n\tat jdk.internal.reflect.GeneratedMethodAccessor241.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)\n\tat com.sun.ejb.containers.interceptors.InterceptorManager.intercept(InterceptorManager.java:375)\n\tat com.sun.ejb.containers.BaseContainer.__intercept(BaseContainer.java:4807)\n\tat com.sun.ejb.containers.BaseContainer.intercept(BaseContainer.java:4795)\n\tat com.sun.ejb.containers.EJBLocalObjectInvocationHandler.invoke(EJBLocalObjectInvocationHandler.java:212)\n\tat com.sun.ejb.containers.EJBLocalObjectInvocationHandlerDelegate.invoke(EJBLocalObjectInvocationHandlerDelegate.java:90)\n\tat com.sun.proxy.$Proxy905.doFilter(Unknown Source)\n\tat de.symeda.sormas.ui.__EJB31_Generated__SessionFilterBean__Intf____Bean__.doFilter(Unknown Source)\n\tat de.symeda.sormas.ui.SessionFilter.doFilter(SessionFilter.java:66)\n\tat org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:253)\n\tat org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:211)\n\tat org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:257)\n\tat org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:160)\n\tat org.apache.catalina.core.StandardPipeline.doInvoke(StandardPipeline.java:757)\n\tat org.apache.catalina.core.StandardPipeline.doChainInvoke(StandardPipeline.java:581)\n\tat com.sun.enterprise.web.WebPipeline.invoke(WebPipeline.java:97)\n\tat org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:158)\n\tat org.apache.catalina.connector.CoyoteAdapter.doService(CoyoteAdapter.java:371)\n\tat org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:238)\n\tat com.sun.enterprise.v3.services.impl.ContainerMapper$HttpHandlerCallable.call(ContainerMapper.java:520)\n\tat com.sun.enterprise.v3.services.impl.ContainerMapper.service(ContainerMapper.java:217)\n\tat org.glassfish.grizzly.http.server.HttpHandler.runService(HttpHandler.java:182)\n\tat org.glassfish.grizzly.http.server.HttpHandler.doHandle(HttpHandler.java:156)\n\tat org.glassfish.grizzly.http.server.HttpServerFilter.handleRead(HttpServerFilter.java:218)\n\tat org.glassfish.grizzly.filterchain.ExecutorResolver$9.execute(ExecutorResolver.java:95)\n\tat org.glassfish.grizzly.filterchain.DefaultFilterChain.executeFilter(DefaultFilterChain.java:260)\n\tat org.glassfish.grizzly.filterchain.DefaultFilterChain.executeChainPart(DefaultFilterChain.java:177)\n\tat org.glassfish.grizzly.filterchain.DefaultFilterChain.execute(DefaultFilterChain.java:109)\n\tat org.glassfish.grizzly.filterchain.DefaultFilterChain.process(DefaultFilterChain.java:88)\n\tat org.glassfish.grizzly.ProcessorExecutor.execute(ProcessorExecutor.java:53)\n\tat org.glassfish.grizzly.nio.transport.TCPNIOTransport.fireIOEvent(TCPNIOTransport.java:524)\n\tat org.glassfish.grizzly.strategies.AbstractIOStrategy.fireIOEvent(AbstractIOStrategy.java:89)\n\tat org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.run0(WorkerThreadIOStrategy.java:94)\n\tat org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.access$100(WorkerThreadIOStrategy.java:33)\n\tat org.glassfish.grizzly.strategies.WorkerThreadIOStrategy$WorkerThreadRunnable.run(WorkerThreadIOStrategy.java:114)\n\tat org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.doWork(AbstractThreadPool.java:569)\n\tat org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.run(AbstractThreadPool.java:549)\n\tat java.base/java.lang.Thread.run(Thread.java:829)\nCaused by: java.lang.NullPointerException\n"}}
{"Timestamp":"2022-09-12T15:28:10.121+0200","Level":"SCHWERWIEGEND","Version":"Payara 5.2021.10","LoggerName":"com.vaadin.server.DefaultErrorHandler","ThreadID":"36040","ThreadName":"http-thread-pool::http-listener-1(21)","TimeMillis":"1662989290121","LevelValue":"1000","Throwable":{"StackTrace":"javax.ejb.EJBTransactionRolledbackException\n\tat de.symeda.sormas.api.share._ExternalShareInfoFacade_Wrapper.getIndexList(de/symeda/sormas/api/share/_ExternalShareInfoFacade_Wrapper.java)\n\tat de.symeda.sormas.ui.externalsurveillanceservice.ExternalShareInfoList.reload(ExternalShareInfoList.java:57)\n\tat de.symeda.sormas.ui.externalsurveillanceservice.ExternalSurveillanceShareComponent.createShareInfoList(ExternalSurveillanceShareComponent.java:94)\n\tat de.symeda.sormas.ui.externalsurveillanceservice.ExternalSurveillanceShareComponent.initLayout(ExternalSurveillanceShareComponent.java:60)\n\tat de.symeda.sormas.ui.externalsurveillanceservice.ExternalSurveillanceShareComponent.<init>(ExternalSurveillanceShareComponent.java:45)\n\tat de.symeda.sormas.ui.externalsurveillanceservice.ExternalSurveillanceServiceGateway.addComponentToLayout(ExternalSurveillanceServiceGateway.java:98)\n\tat de.symeda.sormas.ui.externalsurveillanceservice.ExternalSurveillanceServiceGateway.addComponentToLayout(ExternalSurveillanceServiceGateway.java:44)\n\tat de.symeda.sormas.ui.caze.CaseDataView.initView(CaseDataView.java:183)\n\tat de.symeda.sormas.ui.caze.AbstractCaseView.enter(AbstractCaseView.java:239)\n\tat com.vaadin.navigator.Navigator.performNavigateTo(Navigator.java:778)\n\tat com.vaadin.navigator.Navigator.lambda$navigateTo$9a874efd$1(Navigator.java:702)\n\tat com.vaadin.navigator.ViewBeforeLeaveEvent.navigate(ViewBeforeLeaveEvent.java:54)\n\tat de.symeda.sormas.ui.utils.AbstractDetailView.showUnsavedChangesPopup(AbstractDetailView.java:90)\n\tat de.symeda.sormas.ui.utils.AbstractDetailView.beforeLeave(AbstractDetailView.java:83)\n\tat com.vaadin.navigator.Navigator.runAfterLeaveConfirmation(Navigator.java:730)\n\tat com.vaadin.navigator.Navigator.navigateTo(Navigator.java:701)\n\tat com.vaadin.navigator.Navigator.navigateTo(Navigator.java:678)\n\tat de.symeda.sormas.ui.SormasUI.refreshView(SormasUI.java:100)\n\tat de.symeda.sormas.ui.caze.CaseController.saveCase(CaseController.java:578)\n\tat de.symeda.sormas.ui.caze.CaseController.saveCaseWithFacilityChangedPrompt(CaseController.java:1465)\n\tat de.symeda.sormas.ui.caze.CaseController.lambda$getCaseDataEditComponent$17(CaseController.java:906)\n\tat de.symeda.sormas.ui.utils.CommitDiscardWrapperComponent.onCommit(CommitDiscardWrapperComponent.java:693)\n\tat de.symeda.sormas.ui.utils.CommitDiscardWrapperComponent.doCommit(CommitDiscardWrapperComponent.java:536)\n\tat de.symeda.sormas.ui.utils.CommitDiscardWrapperComponent.commit(CommitDiscardWrapperComponent.java:471)\n\tat de.symeda.sormas.ui.utils.CommitDiscardWrapperComponent.commitAndHandle(CommitDiscardWrapperComponent.java:567)\n\tat de.symeda.sormas.ui.utils.CommitDiscardWrapperComponent$1.buttonClick(CommitDiscardWrapperComponent.java:378)\n\tat jdk.internal.reflect.GeneratedMethodAccessor3804.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat com.vaadin.event.ListenerMethod.receiveEvent(ListenerMethod.java:706)\n\tat com.vaadin.event.EventRouter.fireEvent(EventRouter.java:399)\n\tat com.vaadin.event.EventRouter.fireEvent(EventRouter.java:363)\n\tat com.vaadin.server.AbstractClientConnector.fireEvent(AbstractClientConnector.java:1190)\n\tat com.vaadin.ui.Button.fireClick(Button.java:384)\n\tat com.vaadin.ui.Button$1.click(Button.java:57)\n\tat jdk.internal.reflect.GeneratedMethodAccessor4017.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat com.vaadin.server.ServerRpcManager.applyInvocation(ServerRpcManager.java:155)\n\tat com.vaadin.server.ServerRpcManager.applyInvocation(ServerRpcManager.java:116)\n\tat com.vaadin.server.communication.ServerRpcHandler.handleInvocation(ServerRpcHandler.java:442)\n\tat com.vaadin.server.communication.ServerRpcHandler.handleInvocations(ServerRpcHandler.java:407)\n\tat com.vaadin.server.communication.ServerRpcHandler.handleRpc(ServerRpcHandler.java:275)\n\tat com.vaadin.server.communication.UidlRequestHandler.synchronizedHandleRequest(UidlRequestHandler.java:91)\n\tat com.vaadin.server.SynchronizedRequestHandler.handleRequest(SynchronizedRequestHandler.java:40)\n\tat com.vaadin.server.VaadinService.handleRequest(VaadinService.java:1637)\n\tat com.vaadin.server.VaadinServlet.service(VaadinServlet.java:464)\n\tat javax.servlet.http.HttpServlet.service(HttpServlet.java:750)\n\tat org.apache.catalina.core.StandardWrapper.service(StandardWrapper.java:1637)\n\tat org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:331)\n\tat org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:211)\n\tat de.symeda.sormas.ui.Slf4jMdcFilter.doFilter(Slf4jMdcFilter.java:29)\n\tat org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:253)\n\tat org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:211)\n\tat de.symeda.sormas.ui.SessionFilter.lambda$doFilter$2(SessionFilter.java:73)\n\tat de.symeda.sormas.ui.SessionFilterBean.doFilter(SessionFilterBean.java:37)\n\tat jdk.internal.reflect.GeneratedMethodAccessor3754.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.glassfish.ejb.security.application.EJBSecurityManager.runMethod(EJBSecurityManager.java:588)\n\tat org.glassfish.ejb.security.application.EJBSecurityManager.invoke(EJBSecurityManager.java:408)\n\tat com.sun.ejb.containers.BaseContainer.invokeBeanMethod(BaseContainer.java:4835)\n\tat com.sun.ejb.EjbInvocation.invokeBeanMethod(EjbInvocation.java:665)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:834)\n\tat com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)\n\tat com.sun.ejb.containers.interceptors.SystemInterceptorProxy.doCall(SystemInterceptorProxy.java:163)\n\tat com.sun.ejb.containers.interceptors.SystemInterceptorProxy.aroundInvoke(SystemInterceptorProxy.java:140)\n\tat jdk.internal.reflect.GeneratedMethodAccessor251.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)\n\tat com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)\n\tat org.jboss.weld.module.ejb.AbstractEJBRequestScopeActivationInterceptor.aroundInvoke(AbstractEJBRequestScopeActivationInterceptor.java:72)\n\tat org.jboss.weld.module.ejb.SessionBeanInterceptor.aroundInvoke(SessionBeanInterceptor.java:52)\n\tat jdk.internal.reflect.GeneratedMethodAccessor241.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)\n\tat com.sun.ejb.containers.interceptors.InterceptorManager.intercept(InterceptorManager.java:375)\n\tat com.sun.ejb.containers.BaseContainer.__intercept(BaseContainer.java:4807)\n\tat com.sun.ejb.containers.BaseContainer.intercept(BaseContainer.java:4795)\n\tat com.sun.ejb.containers.EJBLocalObjectInvocationHandler.invoke(EJBLocalObjectInvocationHandler.java:212)\n\tat com.sun.ejb.containers.EJBLocalObjectInvocationHandlerDelegate.invoke(EJBLocalObjectInvocationHandlerDelegate.java:90)\n\tat com.sun.proxy.$Proxy905.doFilter(Unknown Source)\n\tat de.symeda.sormas.ui.__EJB31_Generated__SessionFilterBean__Intf____Bean__.doFilter(Unknown Source)\n\tat de.symeda.sormas.ui.SessionFilter.doFilter(SessionFilter.java:66)\n\tat org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:253)\n\tat org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:211)\n\tat org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:257)\n\tat org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:160)\n\tat org.apache.catalina.core.StandardPipeline.doInvoke(StandardPipeline.java:757)\n\tat org.apache.catalina.core.StandardPipeline.doChainInvoke(StandardPipeline.java:581)\n\tat com.sun.enterprise.web.WebPipeline.invoke(WebPipeline.java:97)\n\tat org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:158)\n\tat org.apache.catalina.connector.CoyoteAdapter.doService(CoyoteAdapter.java:371)\n\tat org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:238)\n\tat com.sun.enterprise.v3.services.impl.ContainerMapper$HttpHandlerCallable.call(ContainerMapper.java:520)\n\tat com.sun.enterprise.v3.services.impl.ContainerMapper.service(ContainerMapper.java:217)\n\tat org.glassfish.grizzly.http.server.HttpHandler.runService(HttpHandler.java:182)\n\tat org.glassfish.grizzly.http.server.HttpHandler.doHandle(HttpHandler.java:156)\n\tat org.glassfish.grizzly.http.server.HttpServerFilter.handleRead(HttpServerFilter.java:218)\n\tat org.glassfish.grizzly.filterchain.ExecutorResolver$9.execute(ExecutorResolver.java:95)\n\tat org.glassfish.grizzly.filterchain.DefaultFilterChain.executeFilter(DefaultFilterChain.java:260)\n\tat org.glassfish.grizzly.filterchain.DefaultFilterChain.executeChainPart(DefaultFilterChain.java:177)\n\tat org.glassfish.grizzly.filterchain.DefaultFilterChain.execute(DefaultFilterChain.java:109)\n\tat org.glassfish.grizzly.filterchain.DefaultFilterChain.process(DefaultFilterChain.java:88)\n\tat org.glassfish.grizzly.ProcessorExecutor.execute(ProcessorExecutor.java:53)\n\tat org.glassfish.grizzly.nio.transport.TCPNIOTransport.fireIOEvent(TCPNIOTransport.java:524)\n\tat org.glassfish.grizzly.strategies.AbstractIOStrategy.fireIOEvent(AbstractIOStrategy.java:89)\n\tat org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.run0(WorkerThreadIOStrategy.java:94)\n\tat org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.access$100(WorkerThreadIOStrategy.java:33)\n\tat org.glassfish.grizzly.strategies.WorkerThreadIOStrategy$WorkerThreadRunnable.run(WorkerThreadIOStrategy.java:114)\n\tat org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.doWork(AbstractThreadPool.java:569)\n\tat org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.run(AbstractThreadPool.java:549)\n\tat java.base/java.lang.Thread.run(Thread.java:829)\nCaused by: javax.transaction.TransactionRolledbackException: CORBA TRANSACTION_ROLLEDBACK 9998 Maybe; nested exception is: \n\torg.omg.CORBA.TRANSACTION_ROLLEDBACK: ----------BEGIN server-side stack trace----------\norg.omg.CORBA.TRANSACTION_ROLLEDBACK:   vmcid: 0x2000  minor code: 1806 completed: Maybe\n\tat org.glassfish.enterprise.iiop.impl.POAProtocolMgr.mapException(POAProtocolMgr.java:356)\n\tat com.sun.ejb.containers.BaseContainer.mapRemoteException(BaseContainer.java:2357)\n\tat com.sun.ejb.containers.BaseContainer.mapRemoteException(BaseContainer.java:2330)\n\tat com.sun.ejb.containers.BaseContainer.postInvoke(BaseContainer.java:2173)\n\tat com.sun.ejb.containers.BaseContainer.postInvoke(BaseContainer.java:2104)\n\tat com.sun.ejb.containers.EJBObjectInvocationHandler.invoke(EJBObjectInvocationHandler.java:212)\n\tat com.sun.ejb.containers.EJBObjectInvocationHandlerDelegate.invoke(EJBObjectInvocationHandlerDelegate.java:79)\n\tat com.sun.proxy.$Proxy686.getIndexList(Unknown Source)\n\tat jdk.internal.reflect.GeneratedMethodAccessor4799.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat com.sun.corba.ee.impl.presentation.rmi.ReflectiveTie.dispatchToMethod(ReflectiveTie.java:143)\n\tat com.sun.corba.ee.impl.presentation.rmi.ReflectiveTie._invoke(ReflectiveTie.java:173)\n\tat com.sun.corba.ee.impl.protocol.ServerRequestDispatcherImpl.dispatchToServant(ServerRequestDispatcherImpl.java:528)\n\tat com.sun.corba.ee.impl.protocol.ServerRequestDispatcherImpl.dispatch(ServerRequestDispatcherImpl.java:199)\n\tat com.sun.corba.ee.impl.protocol.MessageMediatorImpl.handleRequestRequest(MessageMediatorImpl.java:1565)\n\tat com.sun.corba.ee.impl.protocol.SharedCDRClientRequestDispatcherImpl.marshalingComplete(SharedCDRClientRequestDispatcherImpl.java:128)\n\tat com.sun.corba.ee.impl.protocol.ClientDelegateImpl.invoke(ClientDelegateImpl.java:258)\n\tat com.sun.corba.ee.impl.presentation.rmi.StubInvocationHandlerImpl.privateInvoke(StubInvocationHandlerImpl.java:188)\n\tat com.sun.corba.ee.impl.presentation.rmi.StubInvocationHandlerImpl.invoke(StubInvocationHandlerImpl.java:140)\n\tat com.sun.corba.ee.impl.presentation.rmi.codegen.CodegenStubBase.invoke(CodegenStubBase.java:226)\n\tat de.symeda.sormas.api.share.__ExternalShareInfoFacade_Remote_DynamicStub.getIndexList(de/symeda/sormas/api/share/__ExternalShareInfoFacade_Remote_DynamicStub.java)\n\tat de.symeda.sormas.api.share._ExternalShareInfoFacade_Wrapper.getIndexList(de/symeda/sormas/api/share/_ExternalShareInfoFacade_Wrapper.java)\n\tat de.symeda.sormas.ui.externalsurveillanceservice.ExternalShareInfoList.reload(ExternalShareInfoList.java:57)\n\tat de.symeda.sormas.ui.externalsurveillanceservice.ExternalSurveillanceShareComponent.createShareInfoList(ExternalSurveillanceShareComponent.java:94)\n\tat de.symeda.sormas.ui.externalsurveillanceservice.ExternalSurveillanceShareComponent.initLayout(ExternalSurveillanceShareComponent.java:60)\n\tat de.symeda.sormas.ui.externalsurveillanceservice.ExternalSurveillanceShareComponent.<init>(ExternalSurveillanceShareComponent.java:45)\n\tat de.symeda.sormas.ui.externalsurveillanceservice.ExternalSurveillanceServiceGateway.addComponentToLayout(ExternalSurveillanceServiceGateway.java:98)\n\tat de.symeda.sormas.ui.externalsurveillanceservice.ExternalSurveillanceServiceGateway.addComponentToLayout(ExternalSurveillanceServiceGateway.java:44)\n\tat de.symeda.sormas.ui.caze.CaseDataView.initView(CaseDataView.java:183)\n\tat de.symeda.sormas.ui.caze.AbstractCaseView.enter(AbstractCaseView.java:239)\n\tat com.vaadin.navigator.Navigator.performNavigateTo(Navigator.java:778)\n\tat com.vaadin.navigator.Navigator.lambda$navigateTo$9a874efd$1(Navigator.java:702)\n\tat com.vaadin.navigator.ViewBeforeLeaveEvent.navigate(ViewBeforeLeaveEvent.java:54)\n\tat de.symeda.sormas.ui.utils.AbstractDetailView.showUnsavedChangesPopup(AbstractDetailView.java:90)\n\tat de.symeda.sormas.ui.utils.AbstractDetailView.beforeLeave(AbstractDetailView.java:83)\n\tat com.vaadin.navigator.Navigator.runAfterLeaveConfirmation(Navigator.java:730)\n\tat com.vaadin.navigator.Navigator.navigateTo(Navigator.java:701)\n\tat com.vaadin.navigator.Navigator.navigateTo(Navigator.java:678)\n\tat de.symeda.sormas.ui.SormasUI.refreshView(SormasUI.java:100)\n\tat de.symeda.sormas.ui.caze.CaseController.saveCase(CaseController.java:578)\n\tat de.symeda.sormas.ui.caze.CaseController.saveCaseWithFacilityChangedPrompt(CaseController.java:1465)\n\tat de.symeda.sormas.ui.caze.CaseController.lambda$getCaseDataEditComponent$17(CaseController.java:906)\n\tat de.symeda.sormas.ui.utils.CommitDiscardWrapperComponent.onCommit(CommitDiscardWrapperComponent.java:693)\n\tat de.symeda.sormas.ui.utils.CommitDiscardWrapperComponent.doCommit(CommitDiscardWrapperComponent.java:536)\n\tat de.symeda.sormas.ui.utils.CommitDiscardWrapperComponent.commit(CommitDiscardWrapperComponent.java:471)\n\tat de.symeda.sormas.ui.utils.CommitDiscardWrapperComponent.commitAndHandle(CommitDiscardWrapperComponent.java:567)\n\tat de.symeda.sormas.ui.utils.CommitDiscardWrapperComponent$1.buttonClick(CommitDiscardWrapperComponent.java:378)\n\tat jdk.internal.reflect.GeneratedMethodAccessor3804.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat com.vaadin.event.ListenerMethod.receiveEvent(ListenerMethod.java:706)\n\tat com.vaadin.event.EventRouter.fireEvent(EventRouter.java:399)\n\tat com.vaadin.event.EventRouter.fireEvent(EventRouter.java:363)\n\tat com.vaadin.server.AbstractClientConnector.fireEvent(AbstractClientConnector.java:1190)\n\tat com.vaadin.ui.Button.fireClick(Button.java:384)\n\tat com.vaadin.ui.Button$1.click(Button.java:57)\n\tat jdk.internal.reflect.GeneratedMethodAccessor4017.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat com.vaadin.server.ServerRpcManager.applyInvocation(ServerRpcManager.java:155)\n\tat com.vaadin.server.ServerRpcManager.applyInvocation(ServerRpcManager.java:116)\n\tat com.vaadin.server.communication.ServerRpcHandler.handleInvocation(ServerRpcHandler.java:442)\n\tat com.vaadin.server.communication.ServerRpcHandler.handleInvocations(ServerRpcHandler.java:407)\n\tat com.vaadin.server.communication.ServerRpcHandler.handleRpc(ServerRpcHandler.java:275)\n\tat com.vaadin.server.communication.UidlRequestHandler.synchronizedHandleRequest(UidlRequestHandler.java:91)\n\tat com.vaadin.server.SynchronizedRequestHandler.handleRequest(SynchronizedRequestHandler.java:40)\n\tat com.vaadin.server.VaadinService.handleRequest(VaadinService.java:1637)\n\tat com.vaadin.server.VaadinServlet.service(VaadinServlet.java:464)\n\tat javax.servlet.http.HttpServlet.service(HttpServlet.java:750)\n\tat org.apache.catalina.core.StandardWrapper.service(StandardWrapper.java:1637)\n\tat org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:331)\n\tat org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:211)\n\tat de.symeda.sormas.ui.Slf4jMdcFilter.doFilter(Slf4jMdcFilter.java:29)\n\tat org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:253)\n\tat org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:211)\n\tat de.symeda.sormas.ui.SessionFilter.lambda$doFilter$2(SessionFilter.java:73)\n\tat de.symeda.sormas.ui.SessionFilterBean.doFilter(SessionFilterBean.java:37)\n\tat jdk.internal.reflect.GeneratedMethodAccessor3754.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.glassfish.ejb.security.application.EJBSecurityManager.runMethod(EJBSecurityManager.java:588)\n\tat org.glassfish.ejb.security.application.EJBSecurityManager.invoke(EJBSecurityManager.java:408)\n\tat com.sun.ejb.containers.BaseContainer.invokeBeanMethod(BaseContainer.java:4835)\n\tat com.sun.ejb.EjbInvocation.invokeBeanMethod(EjbInvocation.java:665)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:834)\n\tat com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)\n\tat com.sun.ejb.containers.interceptors.SystemInterceptorProxy.doCall(SystemInterceptorProxy.java:163)\n\tat com.sun.ejb.containers.interceptors.SystemInterceptorProxy.aroundInvoke(SystemInterceptorProxy.java:140)\n\tat jdk.internal.reflect.GeneratedMethodAccessor251.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)\n\tat com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)\n\tat org.jboss.weld.module.ejb.AbstractEJBRequestScopeActivationInterceptor.aroundInvoke(AbstractEJBRequestScopeActivationInterceptor.java:72)\n\tat org.jboss.weld.module.ejb.SessionBeanInterceptor.aroundInvoke(SessionBeanInterceptor.java:52)\n\tat jdk.internal.reflect.GeneratedMethodAccessor241.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)\n\tat com.sun.ejb.containers.interceptors.InterceptorManager.intercept(InterceptorManager.java:375)\n\tat com.sun.ejb.containers.BaseContainer.__intercept(BaseContainer.java:4807)\n\tat com.sun.ejb.containers.BaseContainer.intercept(BaseContainer.java:4795)\n\tat com.sun.ejb.containers.EJBLocalObjectInvocationHandler.invoke(EJBLocalObjectInvocationHandler.java:212)\n\tat com.sun.ejb.containers.EJBLocalObjectInvocationHandlerDelegate.invoke(EJBLocalObjectInvocationHandlerDelegate.java:90)\n\tat com.sun.proxy.$Proxy905.doFilter(Unknown Source)\n\tat de.symeda.sormas.ui.__EJB31_Generated__SessionFilterBean__Intf____Bean__.doFilter(Unknown Source)\n\tat de.symeda.sormas.ui.SessionFilter.doFilter(SessionFilter.java:66)\n\tat org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:253)\n\tat org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:211)\n\tat org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:257)\n\tat org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:160)\n\tat org.apache.catalina.core.StandardPipeline.doInvoke(StandardPipeline.java:757)\n\tat org.apache.catalina.core.StandardPipeline.doChainInvoke(StandardPipeline.java:581)\n\tat com.sun.enterprise.web.WebPipeline.invoke(WebPipeline.java:97)\n\tat org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:158)\n\tat org.apache.catalina.connector.CoyoteAdapter.doService(CoyoteAdapter.java:371)\n\tat org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:238)\n\tat com.sun.enterprise.v3.services.impl.ContainerMapper$HttpHandlerCallable.call(ContainerMapper.java:520)\n\tat com.sun.enterprise.v3.services.impl.ContainerMapper.service(ContainerMapper.java:217)\n\tat org.glassfish.grizzly.http.server.HttpHandler.runService(HttpHandler.java:182)\n\tat org.glassfish.grizzly.http.server.HttpHandler.doHandle(HttpHandler.java:156)\n\tat org.glassfish.grizzly.http.server.HttpServerFilter.handleRead(HttpServerFilter.java:218)\n\tat org.glassfish.grizzly.filterchain.ExecutorResolver$9.execute(ExecutorResolver.java:95)\n\tat org.glassfish.grizzly.filterchain.DefaultFilterChain.executeFilter(DefaultFilterChain.java:260)\n\tat org.glassfish.grizzly.filterchain.DefaultFilterChain.executeChainPart(DefaultFilterChain.java:177)\n\tat org.glassfish.grizzly.filterchain.DefaultFilterChain.execute(DefaultFilterChain.java:109)\n\tat org.glassfish.grizzly.filterchain.DefaultFilterChain.process(DefaultFilterChain.java:88)\n\tat org.glassfish.grizzly.ProcessorExecutor.execute(ProcessorExecutor.java:53)\n\tat org.glassfish.grizzly.nio.transport.TCPNIOTransport.fireIOEvent(TCPNIOTransport.java:524)\n\tat org.glassfish.grizzly.strategies.AbstractIOStrategy.fireIOEvent(AbstractIOStrategy.java:89)\n\tat org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.run0(WorkerThreadIOStrategy.java:94)\n\tat org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.access$100(WorkerThreadIOStrategy.java:33)\n\tat org.glassfish.grizzly.strategies.WorkerThreadIOStrategy$WorkerThreadRunnable.run(WorkerThreadIOStrategy.java:114)\n\tat org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.doWork(AbstractThreadPool.java:569)\n\tat org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.run(AbstractThreadPool.java:549)\n\tat java.base/java.lang.Thread.run(Thread.java:829)\nCaused by: javax.ejb.TransactionRolledbackLocalException: Exception thrown from bean: java.lang.NullPointerException\n\tat com.sun.ejb.containers.EJBContainerTransactionManager.checkExceptionClientTx(EJBContainerTransactionManager.java:615)\n\tat com.sun.ejb.containers.EJBContainerTransactionManager.postInvokeTx(EJBContainerTransactionManager.java:485)\n\tat com.sun.ejb.containers.BaseContainer.postInvokeTx(BaseContainer.java:4601)\n\tat com.sun.ejb.containers.BaseContainer.postInvoke(BaseContainer.java:2134)\n\t... 135 more\nCaused by: java.lang.NullPointerException\n\n----------END server-side stack trace-------
   ---  vmcid: 0x2000  minor code: 1806 completed: Maybe\n\tat com.sun.corba.ee.impl.javax.rmi.CORBA.Util.mapSystemException(Util.java:277)\n\tat com.sun.corba.ee.impl.presentation.rmi.StubInvocationHandlerImpl.privateInvoke(StubInvocationHandlerImpl.java:201)\n\tat com.sun.corba.ee.impl.presentation.rmi.StubInvocationHandlerImpl.invoke(StubInvocationHandlerImpl.java:140)\n\tat com.sun.corba.ee.impl.presentation.rmi.codegen.CodegenStubBase.invoke(CodegenStubBase.java:226)\n\tat de.symeda.sormas.api.share.__ExternalShareInfoFacade_Remote_DynamicStub.getIndexList(de/symeda/sormas/api/share/__ExternalShareInfoFacade_Remote_DynamicStub.java)\n\t... 117 more\nCaused by: org.omg.CORBA.TRANSACTION_ROLLEDBACK: ----------BEGIN server-side stack trace----------\norg.omg.CORBA.TRANSACTION_ROLLEDBACK:   vmcid: 0x2000  minor code: 1806 completed: Maybe\n\tat org.glassfish.enterprise.iiop.impl.POAProtocolMgr.mapException(POAProtocolMgr.java:356)\n\tat com.sun.ejb.containers.BaseContainer.mapRemoteException(BaseContainer.java:2357)\n\tat com.sun.ejb.containers.BaseContainer.mapRemoteException(BaseContainer.java:2330)\n\tat com.sun.ejb.containers.BaseContainer.postInvoke(BaseContainer.java:2173)\n\tat com.sun.ejb.containers.BaseContainer.postInvoke(BaseContainer.java:2104)\n\tat com.sun.ejb.containers.EJBObjectInvocationHandler.invoke(EJBObjectInvocationHandler.java:212)\n\tat com.sun.ejb.containers.EJBObjectInvocationHandlerDelegate.invoke(EJBObjectInvocationHandlerDelegate.java:79)\n\tat com.sun.proxy.$Proxy686.getIndexList(Unknown Source)\n\tat jdk.internal.reflect.GeneratedMethodAccessor4799.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat com.sun.corba.ee.impl.presentation.rmi.ReflectiveTie.dispatchToMethod(ReflectiveTie.java:143)\n\tat com.sun.corba.ee.impl.presentation.rmi.ReflectiveTie._invoke(ReflectiveTie.java:173)\n\tat com.sun.corba.ee.impl.protocol.ServerRequestDispatcherImpl.dispatchToServant(ServerRequestDispatcherImpl.java:528)\n\tat com.sun.corba.ee.impl.protocol.ServerRequestDispatcherImpl.dispatch(ServerRequestDispatcherImpl.java:199)\n\tat com.sun.corba.ee.impl.protocol.MessageMediatorImpl.handleRequestRequest(MessageMediatorImpl.java:1565)\n\tat com.sun.corba.ee.impl.protocol.SharedCDRClientRequestDispatcherImpl.marshalingComplete(SharedCDRClientRequestDispatcherImpl.java:128)\n\tat com.sun.corba.ee.impl.protocol.ClientDelegateImpl.invoke(ClientDelegateImpl.java:258)\n\tat com.sun.corba.ee.impl.presentation.rmi.StubInvocationHandlerImpl.privateInvoke(StubInvocationHandlerImpl.java:188)\n\tat com.sun.corba.ee.impl.presentation.rmi.StubInvocationHandlerImpl.invoke(StubInvocationHandlerImpl.java:140)\n\tat com.sun.corba.ee.impl.presentation.rmi.codegen.CodegenStubBase.invoke(CodegenStubBase.java:226)\n\tat de.symeda.sormas.api.share.__ExternalShareInfoFacade_Remote_DynamicStub.getIndexList(de/symeda/sormas/api/share/__ExternalShareInfoFacade_Remote_DynamicStub.java)\n\tat de.symeda.sormas.api.share._ExternalShareInfoFacade_Wrapper.getIndexList(de/symeda/sormas/api/share/_ExternalShareInfoFacade_Wrapper.java)\n\tat de.symeda.sormas.ui.externalsurveillanceservice.ExternalShareInfoList.reload(ExternalShareInfoList.java:57)\n\tat de.symeda.sormas.ui.externalsurveillanceservice.ExternalSurveillanceShareComponent.createShareInfoList(ExternalSurveillanceShareComponent.java:94)\n\tat de.symeda.sormas.ui.externalsurveillanceservice.ExternalSurveillanceShareComponent.initLayout(ExternalSurveillanceShareComponent.java:60)\n\tat de.symeda.sormas.ui.externalsurveillanceservice.ExternalSurveillanceShareComponent.<init>(ExternalSurveillanceShareComponent.java:45)\n\tat de.symeda.sormas.ui.externalsurveillanceservice.ExternalSurveillanceServiceGateway.addComponentToLayout(ExternalSurveillanceServiceGateway.java:98)\n\tat de.symeda.sormas.ui.externalsurveillanceservice.ExternalSurveillanceServiceGateway.addComponentToLayout(ExternalSurveillanceServiceGateway.java:44)\n\tat de.symeda.sormas.ui.caze.CaseDataView.initView(CaseDataView.java:183)\n\tat de.symeda.sormas.ui.caze.AbstractCaseView.enter(AbstractCaseView.java:239)\n\tat com.vaadin.navigator.Navigator.performNavigateTo(Navigator.java:778)\n\tat com.vaadin.navigator.Navigator.lambda$navigateTo$9a874efd$1(Navigator.java:702)\n\tat com.vaadin.navigator.ViewBeforeLeaveEvent.navigate(ViewBeforeLeaveEvent.java:54)\n\tat de.symeda.sormas.ui.utils.AbstractDetailView.showUnsavedChangesPopup(AbstractDetailView.java:90)\n\tat de.symeda.sormas.ui.utils.AbstractDetailView.beforeLeave(AbstractDetailView.java:83)\n\tat com.vaadin.navigator.Navigator.runAfterLeaveConfirmation(Navigator.java:730)\n\tat com.vaadin.navigator.Navigator.navigateTo(Navigator.java:701)\n\tat com.vaadin.navigator.Navigator.navigateTo(Navigator.java:678)\n\tat de.symeda.sormas.ui.SormasUI.refreshView(SormasUI.java:100)\n\tat de.symeda.sormas.ui.caze.CaseController.saveCase(CaseController.java:578)\n\tat de.symeda.sormas.ui.caze.CaseController.saveCaseWithFacilityChangedPrompt(CaseController.java:1465)\n\tat de.symeda.sormas.ui.caze.CaseController.lambda$getCaseDataEditComponent$17(CaseController.java:906)\n\tat de.symeda.sormas.ui.utils.CommitDiscardWrapperComponent.onCommit(CommitDiscardWrapperComponent.java:693)\n\tat de.symeda.sormas.ui.utils.CommitDiscardWrapperComponent.doCommit(CommitDiscardWrapperComponent.java:536)\n\tat de.symeda.sormas.ui.utils.CommitDiscardWrapperComponent.commit(CommitDiscardWrapperComponent.java:471)\n\tat de.symeda.sormas.ui.utils.CommitDiscardWrapperComponent.commitAndHandle(CommitDiscardWrapperComponent.java:567)\n\tat de.symeda.sormas.ui.utils.CommitDiscardWrapperComponent$1.buttonClick(CommitDiscardWrapperComponent.java:378)\n\tat jdk.internal.reflect.GeneratedMethodAccessor3804.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat com.vaadin.event.ListenerMethod.receiveEvent(ListenerMethod.java:706)\n\tat com.vaadin.event.EventRouter.fireEvent(EventRouter.java:399)\n\tat com.vaadin.event.EventRouter.fireEvent(EventRouter.java:363)\n\tat com.vaadin.server.AbstractClientConnector.fireEvent(AbstractClientConnector.java:1190)\n\tat com.vaadin.ui.Button.fireClick(Button.java:384)\n\tat com.vaadin.ui.Button$1.click(Button.java:57)\n\tat jdk.internal.reflect.GeneratedMethodAccessor4017.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat com.vaadin.server.ServerRpcManager.applyInvocation(ServerRpcManager.java:155)\n\tat com.vaadin.server.ServerRpcManager.applyInvocation(ServerRpcManager.java:116)\n\tat com.vaadin.server.communication.ServerRpcHandler.handleInvocation(ServerRpcHandler.java:442)\n\tat com.vaadin.server.communication.ServerRpcHandler.handleInvocations(ServerRpcHandler.java:407)\n\tat com.vaadin.server.communication.ServerRpcHandler.handleRpc(ServerRpcHandler.java:275)\n\tat com.vaadin.server.communication.UidlRequestHandler.synchronizedHandleRequest(UidlRequestHandler.java:91)\n\tat com.vaadin.server.SynchronizedRequestHandler.handleRequest(SynchronizedRequestHandler.java:40)\n\tat com.vaadin.server.VaadinService.handleRequest(VaadinService.java:1637)\n\tat com.vaadin.server.VaadinServlet.service(VaadinServlet.java:464)\n\tat javax.servlet.http.HttpServlet.service(HttpServlet.java:750)\n\tat org.apache.catalina.core.StandardWrapper.service(StandardWrapper.java:1637)\n\tat org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:331)\n\tat org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:211)\n\tat de.symeda.sormas.ui.Slf4jMdcFilter.doFilter(Slf4jMdcFilter.java:29)\n\tat org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:253)\n\tat org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:211)\n\tat de.symeda.sormas.ui.SessionFilter.lambda$doFilter$2(SessionFilter.java:73)\n\tat de.symeda.sormas.ui.SessionFilterBean.doFilter(SessionFilterBean.java:37)\n\tat jdk.internal.reflect.GeneratedMethodAccessor3754.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.glassfish.ejb.security.application.EJBSecurityManager.runMethod(EJBSecurityManager.java:588)\n\tat org.glassfish.ejb.security.application.EJBSecurityManager.invoke(EJBSecurityManager.java:408)\n\tat com.sun.ejb.containers.BaseContainer.invokeBeanMethod(BaseContainer.java:4835)\n\tat com.sun.ejb.EjbInvocation.invokeBeanMethod(EjbInvocation.java:665)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:834)\n\tat com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)\n\tat com.sun.ejb.containers.interceptors.SystemInterceptorProxy.doCall(SystemInterceptorProxy.java:163)\n\tat com.sun.ejb.containers.interceptors.SystemInterceptorProxy.aroundInvoke(SystemInterceptorProxy.java:140)\n\tat jdk.internal.reflect.GeneratedMethodAccessor251.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)\n\tat com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)\n\tat org.jboss.weld.module.ejb.AbstractEJBRequestScopeActivationInterceptor.aroundInvoke(AbstractEJBRequestScopeActivationInterceptor.java:72)\n\tat org.jboss.weld.module.ejb.SessionBeanInterceptor.aroundInvoke(SessionBeanInterceptor.java:52)\n\tat jdk.internal.reflect.GeneratedMethodAccessor241.invoke(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)\n\tat com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)\n\tat com.sun.ejb.containers.interceptors.InterceptorManager.intercept(InterceptorManager.java:375)\n\tat com.sun.ejb.containers.BaseContainer.__intercept(BaseContainer.java:4807)\n\tat com.sun.ejb.containers.BaseContainer.intercept(BaseContainer.java:4795)\n\tat com.sun.ejb.containers.EJBLocalObjectInvocationHandler.invoke(EJBLocalObjectInvocationHandler.java:212)\n\tat com.sun.ejb.containers.EJBLocalObjectInvocationHandlerDelegate.invoke(EJBLocalObjectInvocationHandlerDelegate.java:90)\n\tat com.sun.proxy.$Proxy905.doFilter(Unknown Source)\n\tat de.symeda.sormas.ui.__EJB31_Generated__SessionFilterBean__Intf____Bean__.doFilter(Unknown Source)\n\tat de.symeda.sormas.ui.SessionFilter.doFilter(SessionFilter.java:66)\n\tat org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:253)\n\tat org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:211)\n\tat org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:257)\n\tat org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:160)\n\tat org.apache.catalina.core.StandardPipeline.doInvoke(StandardPipeline.java:757)\n\tat org.apache.catalina.core.StandardPipeline.doChainInvoke(StandardPipeline.java:581)\n\tat com.sun.enterprise.web.WebPipeline.invoke(WebPipeline.java:97)\n\tat org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:158)\n\tat org.apache.catalina.connector.CoyoteAdapter.doService(CoyoteAdapter.java:371)\n\tat org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:238)\n\tat com.sun.enterprise.v3.services.impl.ContainerMapper$HttpHandlerCallable.call(ContainerMapper.java:520)\n\tat com.sun.enterprise.v3.services.impl.ContainerMapper.service(ContainerMapper.java:217)\n\tat org.glassfish.grizzly.http.server.HttpHandler.runService(HttpHandler.java:182)\n\tat org.glassfish.grizzly.http.server.HttpHandler.doHandle(HttpHandler.java:156)\n\tat org.glassfish.grizzly.http.server.HttpServerFilter.handleRead(HttpServerFilter.java:218)\n\tat org.glassfish.grizzly.filterchain.ExecutorResolver$9.execute(ExecutorResolver.java:95)\n\tat org.glassfish.grizzly.filterchain.DefaultFilterChain.executeFilter(DefaultFilterChain.java:260)\n\tat org.glassfish.grizzly.filterchain.DefaultFilterChain.executeChainPart(DefaultFilterChain.java:177)\n\tat org.glassfish.grizzly.filterchain.DefaultFilterChain.execute(DefaultFilterChain.java:109)\n\tat org.glassfish.grizzly.filterchain.DefaultFilterChain.process(DefaultFilterChain.java:88)\n\tat org.glassfish.grizzly.ProcessorExecutor.execute(ProcessorExecutor.java:53)\n\tat org.glassfish.grizzly.nio.transport.TCPNIOTransport.fireIOEvent(TCPNIOTransport.java:524)\n\tat org.glassfish.grizzly.strategies.AbstractIOStrategy.fireIOEvent(AbstractIOStrategy.java:89)\n\tat org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.run0(WorkerThreadIOStrategy.java:94)\n\tat org.glassfish.grizzly.strategies.WorkerThreadIOStrategy.access$100(WorkerThreadIOStrategy.java:33)\n\tat org.glassfish.grizzly.strategies.WorkerThreadIOStrategy$WorkerThreadRunnable.run(WorkerThreadIOStrategy.java:114)\n\tat org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.doWork(AbstractThreadPool.java:569)\n\tat org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.run(AbstractThreadPool.java:549)\n\tat java.base/java.lang.Thread.run(Thread.java:829)\nCaused by: javax.ejb.TransactionRolledbackLocalException: Exception thrown from bean: java.lang.NullPointerException\n\tat com.sun.ejb.containers.EJBContainerTransactionManager.checkExceptionClientTx(EJBContainerTransactionManager.java:615)\n\tat com.sun.ejb.containers.EJBContainerTransactionManager.postInvokeTx(EJBContainerTransactionManager.java:485)\n\tat com.sun.ejb.containers.BaseContainer.postInvokeTx(BaseContainer.java:4601)\n\tat com.sun.ejb.containers.BaseContainer.postInvoke(BaseContainer.java:2134)\n\t... 135 more\nCaused by: java.lang.NullPointerException\n\n----------END server-side stack trace----------  vmcid: 0x2000  minor code: 1806 completed: Maybe\n\tat jdk.internal.reflect.GeneratedConstructorAccessor8266.newInstance(Unknown Source)\n\tat java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)\n\tat java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)\n\tat com.sun.corba.ee.impl.protocol.giopmsgheaders.MessageBase.getSystemException(MessageBase.java:813)\n\tat com.sun.corba.ee.impl.protocol.giopmsgheaders.ReplyMessage_1_2.getSystemException(ReplyMessage_1_2.java:131)\n\tat com.sun.corba.ee.impl.protocol.MessageMediatorImpl.getSystemExceptionReply(MessageMediatorImpl.java:610)\n\tat com.sun.corba.ee.impl.protocol.ClientRequestDispatcherImpl.processResponse(ClientRequestDispatcherImpl.java:506)\n\tat com.sun.corba.ee.impl.protocol.SharedCDRClientRequestDispatcherImpl.marshalingComplete(SharedCDRClientRequestDispatcherImpl.java:166)\n\tat com.sun.corba.ee.impl.protocol.ClientDelegateImpl.invoke(ClientDelegateImpl.java:258)\n\tat com.sun.corba.ee.impl.presentation.rmi.StubInvocationHandlerImpl.privateInvoke(StubInvocationHandlerImpl.java:188)\n\t... 120 more\n"}}
```

--- a/sormas-backend/src/main/java/de/symeda/sormas/backend/share/ExternalShareInfo.java
+++ b/sormas-backend/src/main/java/de/symeda/sormas/backend/share/ExternalShareInfo.java
@@ -66,7 +66,7 @@ public class ExternalShareInfo extends AbstractDomainObject {
 	}
 
 	@ManyToOne
-	@JoinColumn(nullable = false)
+	@JoinColumn
 	public User getSender() {
 		return sender;
 	}

--- a/sormas-backend/src/main/java/de/symeda/sormas/backend/share/ExternalShareInfoFacadeEjb.java
+++ b/sormas-backend/src/main/java/de/symeda/sormas/backend/share/ExternalShareInfoFacadeEjb.java
@@ -34,6 +34,7 @@ import de.symeda.sormas.api.share.ExternalShareInfoCriteria;
 import de.symeda.sormas.api.share.ExternalShareInfoDto;
 import de.symeda.sormas.api.share.ExternalShareInfoFacade;
 import de.symeda.sormas.backend.common.CriteriaBuilderHelper;
+import de.symeda.sormas.backend.user.UserFacadeEjb;
 import de.symeda.sormas.backend.user.UserService;
 import de.symeda.sormas.backend.util.DtoHelper;
 import de.symeda.sormas.backend.util.ModelConstants;
@@ -93,7 +94,7 @@ public class ExternalShareInfoFacadeEjb implements ExternalShareInfoFacade {
 
 		DtoHelper.fillDto(target, source);
 
-		target.setSender(source.getSender().toReference());
+		target.setSender(UserFacadeEjb.toReferenceDto(source.getSender()));
 		target.setStatus(source.getStatus());
 
 		return target;

--- a/sormas-ui/src/main/java/de/symeda/sormas/ui/externalsurveillanceservice/ExternalShareInfoList.java
+++ b/sormas-ui/src/main/java/de/symeda/sormas/ui/externalsurveillanceservice/ExternalShareInfoList.java
@@ -31,6 +31,7 @@ import de.symeda.sormas.api.i18n.Strings;
 import de.symeda.sormas.api.share.ExternalShareInfoCriteria;
 import de.symeda.sormas.api.share.ExternalShareInfoDto;
 import de.symeda.sormas.api.share.ExternalShareStatus;
+import de.symeda.sormas.api.user.UserReferenceDto;
 import de.symeda.sormas.api.utils.DateHelper;
 import de.symeda.sormas.ui.utils.CssStyles;
 import de.symeda.sormas.ui.utils.PaginationList;
@@ -93,8 +94,11 @@ public class ExternalShareInfoList extends PaginationList<ExternalShareInfoDto> 
 			infoLayout.setTemplateContents(buildLayout(shareInfo.getStatus()));
 			infoLayout.setWidth(100, Unit.PERCENTAGE);
 
+			UserReferenceDto sender = shareInfo.getSender();
 			Label senderLabel = new Label(
-				shareInfo.isPseudonymized() ? I18nProperties.getCaption(Captions.inaccessibleValue) : shareInfo.getSender().getShortCaption());
+				shareInfo.isPseudonymized()
+					? I18nProperties.getCaption(Captions.inaccessibleValue)
+					: sender != null ? sender.getShortCaption() : I18nProperties.getCaption(Captions.system));
 			senderLabel.addStyleName(CssStyles.LABEL_BOLD);
 			if (shareInfo.isPseudonymized()) {
 				senderLabel.addStyleName(CssStyles.INACCESSIBLE_LABEL);

